client/src/components/ActionMenu/ActionMenu.js

```diff
@@ -53,6 +53,13 @@ class ActionMenu extends PureComponent {
 
 ActionMenu.defaultProps = {
   className: '',
+  dropdownToggleClassName: classnames(
```

Comment:
I'd suggest we make this an array of strings rather than calling classnames twice